<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWidget</class>
 <widget class="QWidget" name="FilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="filterEdit">
       <property name="placeholderText">
        <string>Filter...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="extensionButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="arrow-down-double"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="extension" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="sortTypeCombo"/>
      </item>
      <item>
       <widget class="QToolButton" name="ascendingButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="arrow-up"/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">sortOrderGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="descendingButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="arrow-down"/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">sortOrderGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="filterCombo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>extensionButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>extension</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="sortOrderGroup"/>
 </buttongroups>
</ui>
